### ISSUE TITLE
ERR_INVALID_URL

### ISSUE THREAD

**Issue by Stregone1** [lightpohl/podcast-dl#50]
I get this error when trying to download this podcast. I get the same (or maybe just similar?) error if I do not save metadata. This is on windows, podcast-dl version is 8.0.5.
```
PS T:\Hoard\Podcasts> .\podcast-dl --archive --include-meta --include-episode-meta --include-episode-images --episode-digits 3 --url "https://www.giantbomb.com/feeds/podcast/"
Giant Bombcast
Giant Bomb discusses the latest video game news and new releases, taste-test questionable beverages, and get wildly off-topic in this weekly podcast.


Downloading podcast image...
https://www.giantbomb.com/a/uploads/original/15/154673/3389479-1461983092-33823.jpg | Starting download of 3.01 MB...
https://www.giantbomb.com/a/uploads/original/15/154673/3389479-1461983092-33823.jpg | Download complete!

Saving podcast metadata...
(node:20836) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_URL]: Invalid URL:
    at new NodeError (internal/errors.js:322:7)
    at onParseError (internal/url.js:270:9)
    at new URL (internal/url.js:346:5)
    at R (C:\snapshot\dist\podcast-dl-8.0.5.js)
    at j (C:\snapshot\dist\podcast-dl-8.0.5.js)
    at $ (C:\snapshot\dist\podcast-dl-8.0.5.js)
    at C:\snapshot\dist\podcast-dl-8.0.5.js
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
(Use `podcast-dl --trace-warnings ...` to show where the warning was created)
(node:20836) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:20836) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by lightpohl** [comment]
Hey! Thanks for the report @Stregone1.

Looks like in the swap over to a Fandom feed they removed all file extensions from the mp3 files. I'll work on adding a fallback for files like these. 👍

**Comment by lightpohl** [comment]
This is now resolved in [v8.0.6](https://github.com/lightpohl/podcast-dl/releases/tag/v8.0.6)!

Thanks again for the report @Stregone1 — let me know if you run into any more issues!

**Comment by Stregone1** [comment]
Appears to be working great now! 👍
